File: src/main/java/org/jukeboxmc/item/behavior/ItemDriedKelp.java
package org.jukeboxmc.item.behavior;

import org.jukeboxmc.item.ItemType;
import org.jukeboxmc.util.Identifier;

/**
 * @author LucGamesYT
 * @version 1.0
 */
public class ItemDriedKelp extends ItemFood {

    public ItemDriedKelp( Identifier identifier ) {
        super( identifier );
    }

    public ItemDriedKelp( ItemType itemType ) {
        super( itemType );
    }

    @Override
    public float getSaturation() {
        return 0.2f;
    }

    @Override
    public int getHunger() {
        return 1;
    }
}

File: JukeboxMC-Server/src/main/kotlin/org/jukeboxmc/server/item/behavior/ItemDriedKelp.kt
package org.jukeboxmc.server.item.behavior

import org.jukeboxmc.api.item.ItemType

class ItemDriedKelp(itemType: ItemType, countNetworkId: Boolean) : JukeboxItemFood(itemType, countNetworkId) {

    override fun getSaturation(): Float {
        return 0.2F
    }

    override fun getHunger(): Int {
        return 1
    }
}